ding/utils/data/dataset.py

```diff
@@ -86,6 +95,18 @@ def mean(self):
     def std(self):
         return self._std
 
+    @property
+    def action_bounds(self) -> np.ndarray:
+        return self._action_bounds
+
+    @property
+    def statistics(self) -> dict:
+        return EasyDict(
```

Comment:
maybe dataclass is better, you can refer to [here](https://github.com/opendilab/DI-engine/blob/main/ding/data/buffer/buffer.py#L41)

Reply:
changed